specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,539 @@
+    ///     the contract doesn't support force unregistration.
+    /// MUST require exactly 1 yoctoNEAR attached balance to prevent restricted function-call access-key call
+    /// (UX wallet security)
+    /// Returns `true` iff the account was unregistered.
+    /// Returns `false` iff account was not registered before.
+    fn storage_unregister(&mut self, token_ids:Vec<TokenId>, force: Option<bool>) -> Vec<bool>;
+
+    fn storage_balance_bounds(&self, token_id:Vec<TokenId>, account_id: Option<AccountId>) -> StorageBalanceBounds;
+    fn storage_balance_of(&self, token_ids:Vec<TokenId>, account_id: AccountId) -> Option<StorageBalance>;
+}
+```
+
+### Metadata Trait
+```
+pub struct MultiTokenMetadata {
+    pub spec: String,              // required, essentially a version like "mt-1.0.0"
+    pub name: String,              // required, ex. "Mosaics"
+    pub symbol: Option<String>,            // ex. "MOSIAC"
+    pub icon: Option<String>,      // Data URL
+    pub base_uri: Option<String>, // Centralized gateway known to have reliable access to decentralized storage assets referenced by `reference` or `media` URLs
+    // supports metadata_uri interface that interpolates {id} in the string
+    pub decimals: Option<u8>, // Option to specify precision if required 
+    pub title: Option<String>, // ex. "Arch Nemesis: Mail Carrier" or "Parcel #5055"
+    pub description: Option<String>, // free-form description
+    pub media: Option<String>, // URL to associated media, preferably to decentralized, content-addressed storage
+    pub media_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of content referenced by the `media` field. Required if `media` is included.
+    pub copies: Option<u64>, // number of copies of this set of metadata in existence when token was minted.
+    pub issued_at: Option<String>, // ISO 8601 datetime when token was issued or minted
+    pub expires_at: Option<String>, // ISO 8601 datetime when token expires
+    pub starts_at: Option<String>, // ISO 8601 datetime when token starts being valid
+    pub updated_at: Option<String>, // ISO 8601 datetime when token was last updated
+    pub extra: Option<String>, // anything extra the NFT wants to store on-chain. Can be stringified JSON.
+    pub reference: Option<String>, // URL to an off-chain JSON file with more info.
+    pub reference_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of JSON from reference field. Required if `reference` is included.
+}
+
+/// Offers details on the  metadata.
+pub trait MultiTokenMetadataProvider {
+    fn mt_metadata(&self, token_id: TokenId) -> MultiTokenMetadata;
+}
+```
+# Drawbacks
+[drawbacks]: #drawbacks
+Doing this adds another spec and codebase to the standards. It could be seen that we could leave this to developers to implement custom solutions and have them create a token that just uses FT and NFT together. There is some additional complexity
```

Comment:
```suggestion
Doing this adds another spec and codebase to the standards. It could be seen that we could leave this to developers to implement custom solutions and have them create a contract that implements both `NEP-141` and `NEP-171` methods together. There is some additional complexity
```